BigQuery/README.md

```diff
@@ -28,13 +28,24 @@ The script `create_ddls.sh` will connect to your database and create a collectio
 The following are the steps to execute the DDL Code Generation. They can be executed in Linux/Unix environments.
 
 1. Modify the `create_ddls.sh` that is located in the `bin` folder
-
-1.1 The region setting will be at the top of this file.
-
-1.2 You must log in going to a link in your browser when you run `./google-cloud-sdk/bin/gcloud init`, and then select the cloud project to use.
-
-2. Finally, run `create_ddls.sh` to extract the DDLs from BigQuery
-
-After a successful run, remove region information from the top line of `create_ddls.sh`.
-
-Compress the entire `Output` folder.
+    - The region setting will be at the top of this file.
+    - You must log in going to a link in your browser when you run `./google-cloud-sdk/bin/gcloud init`, and then select the cloud project to use.
+
+2. Before executing the script ensure `create_ddls.sh` is at the same folder level with `./google-cloud-sdk/`
+    - Finally, run `create_ddls.sh` to extract the DDLs from BigQuery
+    - After a successful run, remove region information from the top line of `create_ddls.sh`.
+
+3.  Compress the entire `Output` folder.
```

Comment:
I know this is not new, but why do we need to compress the output folder? would we remove this?